security.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+permalink: /security/
+---
+## Security Policy
+As Scanly is an open-source project, you are more than welcome to inspect the code for security issues. As well as this, you are able to run any security test against your **own downloaded local copy** of Scanly. **Note** this does not give you permission in anyway to attack the Scanly website, or abuse the demo robot on the site. If you do, you may not only face legal consequences from our group but from our hosting providor.
```

Comment:
```suggestion
As Scanly is an open-source project, you are more than welcome to inspect the code for security issues. As well as this, you are able to run any security test against your **own downloaded local copy** of Scanly. **Note** this does not give you permission in any way to attack the Scanly website, or abuse the demo robot on the site. If you do, you may face legal consequences from us and our infrastructure providers.
```

Reply:
Not commited, as suggestion has already be done above **duplicate message**

Reply:
Aren't these 2 different files though?

Reply:
My bad